packages/calcite-components/src/components/alert/alert.e2e.ts

```diff
@@ -189,6 +202,97 @@ describe("calcite-alert", () => {
     expect(await alert3.isVisible()).toBe(true);
   });
 
+  it("should queue alerts", async () => {
+    const page = await newE2EPage();
+    await skipAnimations(page);
+    await page.setContent(`
+    <div>
+    <calcite-alert id="alert-1">
+    ${alertContent}
+    </calcite-alert>
+    <calcite-alert id="alert-2">
+    ${alertContent}
+    </calcite-alert>
+    <calcite-alert id="alert-3">
+    ${alertContent}
+    </calcite-alert>
+    </div>`);
+
+    const alert1 = await page.find("#alert-1");
+    const alert2 = await page.find("#alert-2");
+    const alert3 = await page.find("#alert-3");
+
+    expect(await alert1.isVisible()).toBe(false);
+    expect(await alert2.isVisible()).toBe(false);
+    expect(await alert3.isVisible()).toBe(false);
+
+    alert1.setProperty("open", true);
+    await page.waitForChanges();
+    alert2.setProperty("open", true);
+    await page.waitForChanges();
+    alert3.setProperty("queue", "immediate");
+    await page.waitForChanges();
+    alert3.setProperty("open", true);
+    await page.waitForChanges();
+    await page.waitForTimeout(animationDurationInMs);
```

Comment:
Can you check if this is needed? https://github.com/Esri/calcite-design-system/pull/9958 fixed some issues that prevented alert (among other components) from honoring `--calcite-duration-factor` (used by `skipAnimations`). If you still need to wait for the alert to be open, you could listen for the open event instead.

Reply:
Yeah this is still needed because the queue uses a setTimeout to wait for the animation to end.

https://github.com/Esri/calcite-design-system/blob/089cecf9a2ff884d76286dced96c5568649a277d/packages/calcite-components/src/components/alert/alert.tsx#L528-L531

I think that above would need to change to pull from the CSS var and convert the string MS value to an integer.

Maybe a good follow up issue?

Reply:
Follow-up issue sounds good. Thanks for checking!

Reply:
https://github.com/Esri/calcite-design-system/issues/10098